ui/src/App.tsx

```diff
@@ -34,14 +36,9 @@ export default function({connection}: Props) {
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [state.identified, state.roomId])
 
-  const disconnectedComponent = () => <div className="App_warning">
-    <div>⚠ Disconnected</div>
-    <div className="indication">Please refresh page</div>
-  </div>
-
   return (
     <div className="App">
-      { !state.connected && disconnectedComponent()}
+      { !state.connected && state.connectedExpected && disconnectedComponent()}
```

Comment:
wondering if a tri-state here "connecting, connected, disconnected" would be clearer... Just an idea though.

Reply:
ok I'll check if this makes sense